packages/office-ui-fabric-react/src/components/Icon/examples/Icon.SvgFactory.Example.tsx

```diff
@@ -0,0 +1,30 @@
+// tslint:disable:jsx-wrap-multiline max-line-length
+import * as React from 'react';
+import { OneDriveIcon, YammerIcon, BorderBlindsIcon } from '@fluentui/react-icons';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const iconClass = mergeStyles({
+  fontSize: 50,
+  height: 50,
+  width: 50,
+  margin: '0 25px',
+});
+
+export const IconSvgFactoryExample: React.FunctionComponent = () => {
+  return (
+    <div>
+      <OneDriveIcon className={iconClass} />
+      <YammerIcon
+        className={mergeStyles(iconClass, {
```

Comment:
You should probably get rid of the inline mergeStyles call--this isn't a good pattern to demonstrate perf-wise. Since this class doesn't depend on anything within the component, you could just move the call to the top of the file.

Reply:
Got rid of classnames selectors and inline overrides.